Source/QED/warpx_schwinger_pair_prod_methods.cpp

```diff
@@ -3,6 +3,56 @@
 #include "WarpX.H"
 #include "schwinger_pair_prod_wrapper.h"
 
-void WarpX::DoSchwingerPairProduction(){
-    //Do nothing for now!
+void WarpX::DoSchwingerPairProduction(amrex::Real a_dt)
+{
+    //Takes the fields of the coarsest grid
```

Comment:
I guess that what we want is to iterate over all the cells of the coarsest level. Does this really do this?

The important thing is in any case to process each region of the physical space exactly once.